pantheon-files-daemon/FileManager1.vala

```diff
@@ -43,7 +43,7 @@ public class FileManager1 : Object {
         string cmd = "pantheon-files -t";
 
         foreach (string s in uris) {
-            cmd += (" " + s);
+            cmd += (" \"" + PF.FileUtils.sanitize_path (s) + "\"");
```

Comment:
I agree that it's something that's unlikely to be easily exploited. But if there's a way to reduce an attack surface by replacing double quotes with single quotes, then we probably should do it.